core/src/main/java/com/datastrato/gravitino/storage/relational/JDBCDatabase.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.storage.relational;
+
+import com.datastrato.gravitino.Config;
+
+public interface JDBCDatabase {
+
+  /**
+   * Initializes the embedded Relational Backend environment with the provided configuration.
+   *
+   * @param config The configuration for the backend.
+   * @throws RuntimeException
+   */
+  void initialize(Config config) throws RuntimeException;
```

Comment:
Do you need a close method? For H2, when do we close the database?

Reply:
It doesn't need to do so as the database is actually within the Java Virtual Machine. I will add it since it is recommended to release resources manually.